packages/font-glyphs/src/auto-build/composite.ptl

```diff
@@ -676,6 +676,10 @@ glyph-block AutoBuild-Enclosure : begin
 		foreach [j : range 36 till 50] : compositions.push : list (0x32B1 + j - 36) [digitGlyphNames j] WideWidth1
 		createCircledGlyphs 2 compositions
 
+	do "Double-digit backslash-circled"
```

Comment:
Consider remove.
If we are not adding keycaps then we should remove the circled 18 with overlaid backslash too.
(Actually, I do NOT want to add ANY MORE new Emoji into Iosevka.)